environment.yml

```diff
@@ -1,20 +1,20 @@
-name: image-analysis-23
+name: image-analysis-24
 channels:
   - conda-forge
 dependencies:
-  - python=3.10
+  - python=3.12
   - pip
   - jupyterlab
   - jupytext
   - notebook
   - matplotlib
-  - scikit-image=0.21.0
-  - jupyter-book=0.15.1
-  - pooch=1.7.0
-  - scikit-learn=1.2.2
-  - ipywidgets=8.0.7
-  - seaborn=0.12.2
-  - napari=0.4.18
+  - scikit-image=0.24.0
+  - jupyter-book=1.0.0
+  - pooch=1.8.2
+  - scikit-learn=1.5.0
+  - ipywidgets=8.1.3
+  - seaborn=0.13.2
+  - napari=0.4.19.post1
   - cookiecutter
   - cellpose
   - pip:
```

Comment:
Not sure why napari-animation is being installed from pip.
it's on conda-forge and it's safer to install that way because ffmpeg.

Reply:
sorry won't let me suggest the fix

Reply:
Not sure why we did it this way during the last tutorial. Maybe the conda-forge one was out of date? I'll update it to conda-forge. :+1: